src/gt4py/next/program_processors/runners/dace_iterator/itir_to_tasklet.py

```diff
@@ -596,3 +600,33 @@ def closure_to_tasklet_sdfg(
     for output in outputs:
         context.body.arrays[output.value.data].transient = False
     return context, outputs
+
+
+def lambda_to_tasklet_sdfg(
+    node: itir.StencilClosure,
+    offset_provider: dict[str, Any],
+    node_types: dict[int, next_typing.Type],
+) -> tuple[Context, Sequence[tuple[str, ValueExpr]], list[ValueExpr]]:
+    body = dace.SDFG("tasklet_toplevel")
+    state = body.add_state("tasklet_toplevel_entry")
+    symbol_map: dict[str, ValueExpr | IteratorExpr] = {}
+
+    context = Context(body, state, symbol_map)
+
+    def get_lambda() -> Lambda:
+        stencil_fobj = cast(FunCall, node.stencil)
+        return cast(Lambda, stencil_fobj.args[0])
+
+    call = itir.Lambda(expr=get_lambda().expr, params=get_lambda().params)
```

Comment:
```suggestion
    callable = itir.Lambda(expr=get_lambda().expr, params=get_lambda().params)
```

Reply:
`A001 variable "callable" is shadowing a python builtin`

Instead, I can use `funobj`, for function object.